src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -361,6 +384,15 @@ private static DownloadDependency ToPackageDownloadDependency(IVsReferenceItem i
             return downloadDependency;
         }
 
+        private static CentralPackageVersion ToCentralPackageVersion(IVsReferenceItem item)
+        {
+            var id = item.Name;
+            var versionRange = GetVersionRange(item);
```

Comment:
I believe [Coding Guideline ](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style)# 10 is being violated.
`var` should not be used for Name or GetVersionRange, as their types are not immediately obvious without jumping to their definitions.

Reply:
Updated.